### DISCUSSION TITLE
Read image from data directory

### DISCUSSION THREAD

**Discussion by juan518munoz** [zepp-health/zeppos-forum#75] [Q&A] [answered]
Is it possible to use the IMG widget with images read by the file system (hmFS)?
https://docs.zepp.com/docs/1.0/guides/framework/device/fs/#introduction

**Comment by alexmathew11** [comment]
I also wanted to know

**Comment by bigdigital** [comment, marked as answer]
You can write images to the "assets" folder and then display the images via IMG  widget. I used such an approach to transfer images via side app 
here is   [code  i used to write the image ](https://github.com/bigdigital/zeppos_watchdrip_app/blob/master/pages/index.js#L356)
please note the image should be written with a modified TGA format https://github.com/bigdigital/zeppos_watchdrip_app/blob/master/shared/tga-img.js
